compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,137 @@
+; To activate a step (or task),
+; set True for the step in [onecc] section and fill options in the corresponding section
 [onecc]
-one-import-tf=True
+; neural network model to circle
+one-import-tf=False
 one-import-tflite=False
 one-import-bcq=False
 one-import-onnx=False
-one-optimize=True
+; circle to circle with optimization
+one-optimize=False
+; circle to circle with quantization
 one-quantize=False
+; partition circle
 one-partition=False
-one-pack=True
+; package circle and metadata into nnpackage
+one-pack=False
+; generate code for backend
 one-codegen=False
+; profile
 one-profile=False
 
 [one-import-tf]
-input_path=/path/to/inception_v3.pb
-output_path=inception_v3.circle
-input_arrays=input
-input_shapes=1,299,299,3
-output_arrays=InceptionV3/Predictions/Reshape_1
-converter_version=v1
-model_format=graph_def
+# mandatory
+; pb file
+input_path=
+; circle file
+output_path=
+# optional
+; v1 or v2
+converter_version=v2
+# optional but mandatory for converter_version=v1
+; input tensor names
+input_arrays=
+; eg) 1,299,299,3
+input_shapes=
+; output tensor names
+output_arrays=
+
+[one-import-tflite]
+# mandatory
+; tflite file
+input_path=
+; circle file
+output_path=
+
+[one-import-bcq]
+# mandatory
+; bcq file
+input_path=
+; circle file
+output_path=
+# optional
+; v1 or v2
+converter_version=v2
+# optional but mandatory for converter_version=v1
+; input tensor names
+input_arrays=
+; eg) 1,299,299,3
+input_shapes=
+; output tensor names
+output_arrays=
+
+[one-import-onnx]
+# mandatory
+; onnx file
+input_path=
+; circle file
+output_path=
+# optional
+; input tensor names
+input_arrays=
+; output tensor names
+output_arrays=
```

Comment:
one-import-onnx will (1) convert to TF v2/saved_model and then (2) run tf2tfliteV2 ...
when v2 is used, input_arrays and output_arrays in tf2tfliteV2 are not used as TF v2 model provides them.
I think you may have referenced current one-import-onnx option but this has some issue and need changes (#9161).

Reply:
Yes exactly. I'll remove them.